happy-birthday-to-you.js

```diff
@@ -0,0 +1,153 @@
+/* 
+
+------ STEPS ------
+1.  Ensure that name is in correct format (only strings, also account for 
+	special characters).
+
+2. Once correct name format is inputted, compare it to object key/value pairs.
+
+2a. Play a birthday song if dates match. otherwise, play the Rick Roll 
+
+*/
+
+const birthdayObj = {
+
+	'Abdullah Sayeem': 513,
+	'Andre Chow': 103,
+	'Gurdeep Singh': 327,
+	'Jimmy Feng': 711,
+	'John Chen': 1202,
+	'Kevin Chen': 627,
+	'Kevin Cheung':	521,
+	'Mohammed Rahman': 305,
+	'Robert Tan': 523,
+	'Roland Zhou': 903,
+	'Vainius Glinskis': 805,
+	'Zarif Rahat': 1107,
+
+};
+
+function validateInputName() {
+	let text = nameInput.value.toLowerCase();
+	// any combination of letters separated by a whitespace
+	let nameFormat = /^[a-zA-Z]+\s+[a-zA-Z]+$/; 
```

Comment:
can be const since you don't change it. Basically always use const until you have to use let.